packages/sdk/src/interaction/interaction.js

```diff
@@ -138,6 +138,16 @@ const makeIx = wat => ix => {
   return Ok(ix)
 }
 
+const prepAccountKeyId = acct => {
+  if (!acct.keyId) return acct
+
+  invariant(Number.isInteger(+acct.keyId), "account.keyId must be an integer")
+  return {
+    ...acct,
+    keyId: +acct.keyId,
```

Comment:
Thoughts on this vs `Number(acct.keyId)` ?

Reply:
Number is maybe more verbose